card/mod/standard/set/all/rich_html/toolbar.rb

```diff
@@ -143,8 +143,8 @@ def nests_editable?
   def account_split_button
     return "" unless card.accountable?
     toolbar_split_button "account", related: :account, icon: :account_box do
-      %i[ account roles created edited follow ].each_with_object({}) do |item, hash|
-        label = item == :account ? "details" : item.to_s # TODO: i18n
+      %i[account roles created edited follow].each_with_object({}) do |item, hash|
+        label = item == :account ? tr(:details) : tr(item)
```

Comment:
@ethn writing your TODO comment was more work than adding i18n

Reply:
ha, I didn't realize the translation was there.